src/test/java/io/personium/core/auth/AccessContextTest.java

```diff
@@ -74,9 +84,17 @@
 
     /**
      * トークン処理ライブラリの初期設定.
+     * @throws IOException 
+     * @throws InvalidNameException 
+     * @throws CertificateException 
+     * @throws InvalidKeySpecException 
+     * @throws Exception 
      */
     @BeforeClass
-    public static void beforeClass() {
+    public static void beforeClass() throws Exception {
+        PersoniumCoreApplication.loadConfig();
+        TransCellAccessToken.configureX509(PersoniumUnitConfig.getX509PrivateKey(),
+                PersoniumUnitConfig.getX509Certificate(), PersoniumUnitConfig.getX509RootCertificate());
```

Comment:
We need these initialization lines since this part used to be handled by the IntegTestRunner.